packages/controllers/src/controllers/ApiController.ts

```diff
@@ -363,5 +363,9 @@ export const ApiController = {
     state.recommended = state.allRecommended.filter(wallet =>
       wallet.chains?.some(chain => caipNetworkIds.includes(chain))
     )
+
+    state.wallets = state.wallets.filter(wallet =>
```

Comment:
Should we clear the filtering on top if namespace is undefined?

Reply:
And after filtering first time, how to remove filter as the state.wallets are filtered before, it doesn have all the wallets 🤔

Reply:
But isn't this reapplied every time the network specific button is clicked?

Reply:
Changed the aproach to filter on fetch instead of here